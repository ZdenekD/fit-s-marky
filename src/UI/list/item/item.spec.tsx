import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../../__test__/utils/helpers';
import Item from '.';

describe('Item', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Item isAnimated={false} />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-item');

        expect(component.exists()).toBe(true);
    });
});
