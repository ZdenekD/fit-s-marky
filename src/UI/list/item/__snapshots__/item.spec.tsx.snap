// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item match snapshot 1`] = `
<li class="item "
    data-test="component-item"
>
</li>
`;
